homeassistant/components/deconz/climate.py

```diff
@@ -17,6 +24,18 @@
 from .deconz_device import DeconzDevice
 from .gateway import get_gateway_from_config_entry
 
+DECONZ_FAN_SMART = "smart"
+
+FAN_MODES = {
+    DECONZ_FAN_SMART: "smart",
+    FAN_AUTO: "auto",
+    FAN_HIGH: "high",
+    FAN_MEDIUM: "medium",
+    FAN_LOW: "low",
+    FAN_ON: "on",
+    FAN_OFF: "off",
```

Comment:
Check **climate/const.py**, there will already defines for low, medium, high, on, off and auto that can be imported instead of the raw strings.

Reply:
Its a translation from hass to deconz values, so it shouldn't use hass constants for deconz